src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -275,6 +277,7 @@ data class CommonAmendmentFields(
   val authorUsername: String,
 ) {
   override fun toString(): String = "{text-hash=${Objects.hashCode(text)}, occurrenceDateTime=$occurrenceDateTime, authorUsername=$authorUsername}"
+  fun toNomisAmendment(): String = " ...[$authorUsername updated the case notes on $occurrenceDateTime] $text"
```

Comment:
we copy the amendment into the case note text so are therefore testing all the amendment fields here anyway